lib/modules/manager/regex/strategies.ts

```diff
@@ -44,7 +44,10 @@ export function handleCombination(
   const extraction = matches
     .map((match) => ({
       groups: match.groups ?? {},
-      replaceString: match?.groups?.currentValue ? match[0] : undefined,
+      replaceString:
+        match?.groups?.currentValue || match?.groups?.currentDigest
```

Comment:
```suggestion
        match?.groups?.currentValue ?? match?.groups?.currentDigest
```
what happens when version and digest are in separate regex matches? would that currently also fail?

Reply:
I believe it's OK due to the `.reduce` function on line 52.